docs/data-sources/connector.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_connector Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's documentation https://docs.twingate.com/docs/understanding-access-nodes.
+---
+
+# twingate_connector (Data Source)
+
+Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/understanding-access-nodes).
+
+## Example Usage
+
+```terraform
+data "twingate_connector" "foo" {
+  id = "<your connector's id>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The ID of the Connector
+
+### Read-Only
+
+- `name` (String) Name of the Connector, if not provided one will be generated
+- `remote_network_id` (String) The ID of the Remote Network attached to the Connector
```

Comment:
Change to:

`The ID of the Remote Network the Connector is attached to`

Reply:
We should change this description for this value in the twingate_connector resource, too.

Reply:
updated